code/mod_table/mod_table.cpp

```diff
@@ -267,6 +285,42 @@ void parse_mod_table(const char *filename)
 			mprintf(("Game Settings Table: $BMPMAN Slot Limit is deprecated and should be removed. It is not needed anymore.\n"));
 		}
 
+		if (optional_string("$EMP Arc Color:")) {
+			if (optional_string("+Primary Color Option 1:")) {
+				stuff_ubyte(&Arc_color_damage_p1_r);
+				stuff_ubyte(&Arc_color_damage_p1_g);
+				stuff_ubyte(&Arc_color_damage_p1_b);
```

Comment:
I think it would be better to use `stuff_int_list` here since that already has an established parser implementation for parsing number lists. You will have to do some additional validation though to make sure that exactly 3 values are specified and that those values are in the correct range.

Reply:
Couldn't figure out a efficient way of using `stuff_int_list` since the tuples have ubytes for values, and DahBlount stated using three values and `stuff_ubyte` would probably work best so I am sticking with that option for now. I did add in the error checks though, good idea!